readme.md

```diff
@@ -1,3 +1,8 @@
+# not net pa now got 
+i wil be available shal we meet
+# -----------
+is every thing ok what need to be done
```

Comment:
no i am waiting for from the morning

Reply:
now cooking will be avaiable at 9.30pm partial avagilable now

Reply:
> no i am waiting for from the morning

sry pa.

Reply:
check all the code new bug than iruku shall move to prod deployment